bindings/ios/MEGARequest.mm

```diff
@@ -193,4 +208,14 @@ - (NSInteger)numDetails {
     return self.megaRequest ? self.megaRequest->getNumDetails() : 0;
 }
 
+- (NSArray<MEGAStringList *> *)megaStringTableArray {
+    MegaStringTable *table = self.megaRequest ? self.megaRequest->getMegaStringTable() : nil;
+    NSMutableArray<MEGAStringList *> *array = NSMutableArray.alloc.init;
+    for (int i = 0; i < table->size(); i++) {
+        [array addObject:[[MEGAStringList alloc] initWithMegaStringList:(MegaStringList *)table->get(i) cMemoryOwn:YES]];
```

Comment:
@jcmartinac, memory leaks here. Please copy the returned `MegaStringList` because we need to take the ownership here. `table->get(i)->copy()`

```suggestion
        [array addObject:[[MEGAStringList alloc] initWithMegaStringList:(MegaStringList *)table->get(i)->copy() cMemoryOwn:YES]];
```

Reply:
Thanks Simon. I have been talking with Navarro and Santana about the naming and the return object to decide returning native classes instead of SDK classes, so I have refactored this method. This change is not needed.

Reply:
Thanks Carlos. Yes, I also think `NSArray<NSString *> *` would be better than `MEGAStringList` for our native code. I can not see your changes. Have you pushed your local changes?

Reply:
Simon, I have just pushed the commit. I wanted to check some things with Sergio before push it.